### DISCUSSION TITLE
settings_ option_ find new settings introduced in latest version

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#12803] [Q&A]
* i am on android
* assuming lot of android users dont read at all changelogs
* is there any option to view new options introduced in latest version
* why?
 * there is new quite interesting option for page turning https://github.com/koreader/koreader/releases/tag/v2024.11
 * my memory is haze, but i found new options marked with red dot in samsung internet app, which is super cool
 * i hope there is an option or similar to samsung internet

**Comment by Commodore64user** [comment]
> * i am on android

okay 

> * assuming lot of android users dont read at all changelogs

everyone is free to do as they wish

> * is there any option to view new options introduced in latest version

Yes, the changelog that accompanies the official release, which you linked below.

> * why?

so people know what has changed.

> * there is new quite interesting option for page turning https://github.com/koreader/koreader/releases/tag/v2024.11

not sure what you are referring to here, but it could be kindle only though (assuming you are referencing the left/right one)

> * my memory is haze, but i found new options marked with red dot in samsung internet app, which is super cool

I'm so sorry about your memory, and I have no clue what the red dot is... sorry

> * i hope there is an option or similar to samsung internet

still no idea.

**Reply by ghost** [reply]
> I'm so sorry about your memory, and

cant help, selective amnesia, some say my memory is as good as crooked politician

> I have no clue what the red dot is... sorry
* i have samsung device
* in app settings, tap uninstall updates
* updated it to latest version
* all new options have red dot
* red dot video

https://github.com/user-attachments/assets/38ef95c1-d736-439a-9989-39841bbadeee

**Reply by Commodore64user** [reply]
Black screen?

**Reply by benoit-pierre** [reply]
Ah! So I'm not the only one. The video works fine in Chromium, but not in Firefox.

**Reply by Frenzie** [reply]
It's an AVC video. It seems Firefox doesn't ship support for copyright reasons.
